Sprint-1/destructuring/exercise-3/exercise.js

```diff
@@ -6,3 +6,19 @@ let order = [
   { itemName: "Hot Coffee", quantity: 2, unitPricePence: 100 },
   { itemName: "Hash Brown", quantity: 4, unitPricePence: 40 },
 ];
+
+
+function orderReceipt(arr){
+  
+  console.log('QTY'.padEnd(8),'ITEM'.padEnd(18),'TOTAL')
+  let totalOrder = 0
+  for(const {quantity, itemName, unitPricePence} of arr){
+    const totalPerItem = ((unitPricePence * quantity) / 100).toFixed(2)
+    totalOrder += unitPricePence * quantity
+    console.log(`${quantity.toString().padEnd(8)}${itemName.padEnd(20)}${totalPerItem}`)
+  }
+
+  console.log(`\nTotal: ${(totalOrder/100).toFixed(2)}`)
```

Comment:
There are a few pieces of repetitive code here (code where you write down how to do the exact same thing twice, where having a name for that operation could also be clarifying) - can you identify and deduplicate them?